.github/workflows/tests-ast.yml

```diff
@@ -14,18 +14,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-            # Build on the oldest and latest supported Pythons
-            # and on the oldest and latest ANTLR version supported
-            # by each of those Pythons:
-            - python-version: '3.9'
-              antlr-version: '4.7'
-            - python-version: '3.9'
-              antlr-version: '4.13'
-            - python-version: '3.13'
-              antlr-version: '4.9'
-            - python-version: '3.13'
-              antlr-version: '4.13'
+        # Just using minimum and maximum to avoid exploding the matrix.
+        python-version: ['3.9', '3.12']
+        antlr-version: ['4.7', '4.13']
```

Comment:
Reminder to myself to just grab the latest changes to this file from the `main` branch.

Reply:
Fixed in [3a94131](https://github.com/openqasm/openqasm/pull/563/commits/3a9413128657d260fcfb1a1c33110f13163584c0)